File: mlproject/Mark.java
public class Mark{
	static final int x=1;
	static final int o=2;
	static final int blank=0;
	static final int getOpposite(int mark)
	{
		if(mark==blank)
			return mark;
		return mark==Mark.x ? Mark.o : Mark.x;
	}
}

File: toPython/Mark.py
class Mark():
    x = 1
    o = 2
    blank = 0

    def toChar(mark):
        if mark == Mark.blank:
            return ' '
        elif mark == Mark.x:
            return 'X'
        else:
            return 'O'

    def getOpposite(mark):
        if mark == Mark.blank:
            return mark
        elif mark == Mark.x:
            return Mark.o
        else:
            return Mark.x
